pom.xml

```diff
@@ -172,6 +172,9 @@
                             <goal>check</goal>
                         </goals>
                         <configuration>
+                            <excludes>
+                                <exclude>com/thebois/**/ItemFactory.class</exclude>
+                            </excludes>
```

Comment:
Question



































Why?

Reply:
Oh, actually, I removed that switch expression. Good catch.

Reply:
👍